docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source

```diff
@@ -0,0 +1,109 @@
+---
+id: aws-security-data-lake-source
+title: AWS Security Data Lake Source (Beta)
+description: This document explains how to configure the AWS Security Data Lake source setup using the Sumo logic environment.
```

Comment:
```suggestion
description: This document explains how to configure the AWS Security Data Lake source setup using the Sumo Logic environment.
```

Reply:
And other occurrences should be reviewed as well

Reply:
@sumo-drosiek : Thanks for pointing out that. I have fixed it in other instances as well.